karabo_data/geometry2.py

```diff
@@ -976,6 +997,38 @@ def split_tiles(module_data):
         # starts at the bottom. So we reverse their order after splitting.
         return np.split(half1, 8, axis=-2)[::-1] + np.split(half2, 8, axis=-2)
 
+    @classmethod
+    def _distortion_array_slice(cls, m, t):
+        # Which part of the array is this tile?
+        # The distortion array for LPD is 4096 x 256, with modules stacked
+        # along their slow-scan axis, Q1M1 (m=0) to Q4M4 (m=15)
+        module_offset = m * 256
+        if t < 8:  # First half of module (0 <= t <= 7)
+            fs_slice = slice(0, 128)
+            tiles_up = 7 - t
+        else:      # Second half of module (8 <= t <= 15)
+            fs_slice = slice(128, 256)
```

Comment:
Pre-emptively, because I'm sure @tmichela will ask: I tried writing this method with `cls.frag_ss_pixels` etc. instead of hardcoded numbers. But I found it was much harder to read when I did that, so I went back to hardcoded numbers. This is on the specific LPD class anyway, so the numbers can't really change.